### PR TITLE
Side feed

### PR DESCRIPTION
**Pull request by alanorwick**

### CODE REVIEW

File: manager/main.go [hunk 1]
```diff
@@ -740,11 +739,9 @@ func (m *Manager) resultLoop() error {
...
 
 			// Check to see that all nodes are running before sending blocks to them.
-			for _, blockClient := range m.orderedBlockClients {
-				if !checkConnection(blockClient.chainClient) {
-					log.Println("Chain unavailable, for URL", blockClient.chainAvailable, "continuing...")
-					continue
-				}
+			if !m.allChainsOnline() {
+				log.Println("At least one of the chains is not online at the moment")
+				continue
```
Comment (alanorwick) [inline manager/main.go:744]: I don't think we should continue here

Comment (alanorwick) [reply manager/main.go:744]: Wait nevermind it's fine cause it's just the hashrate.

Comment (shreekarashastry) [reply manager/main.go:744]: this will probably create issues though, if one of the nodes are not online with external blocks

Comment (shreekarashastry) [reply manager/main.go:744]: that's why I asked that question

**Review by shreekarashastry — APPROVED** [review]

**Pull request merged**